src/main/service.js

```diff
@@ -294,47 +294,45 @@ function deleteUser(req, res, next){
  * return a list of meta question of the user's course
  * @throws {Error} - if fail to fetch
  */
-function getAllMetaQuestions(req, res, next){
-    try{
-        let metaQuestions = application.getAllMetaQuestions(req.body);
-        req.log.info("a request was sent fetch all the meta questions");
-        res.send(200, {code:200, metaQuestions})
-        next()
-    }
-    catch(err){
-        req.log.warn(err.message, 'unable to request fetch all the meta questions');
-        next(err);
-    }
+function getAllMetaQuestions(req, res, next) {
+    application.getAllMetaQuestions(req.body).then(
+        metaQuestions => {
+            req.log.info("a request was sent fetch all the meta questions");
```

Comment:
*irrelevent*
but you can say that the request was successs, no ?

Reply:
I don't even know how this `log` is expressed

Reply:
LAMO what a great system we made